tfhe/src/shortint/engine/server_side/shift.rs

```diff
@@ -20,7 +20,7 @@ impl ShortintEngine {
         ct: &mut Ciphertext,
         shift: u8,
     ) -> EngineResult<()> {
-        let acc = self.generate_lookup_table(server_key, |x| x >> shift)?;
+        let acc = self.generate_msg_lookup_table(server_key, |x| x >> shift, ct.message_modulus)?;
```

Comment:
Yes
If this function was used to build the integer API, this would break things
But I also guess that this function is supposed to be used ignoring the carry bits

Reply:
could be we'll know once you have pushed the updated commit respecting the conventional notaion for approval test runs